index.html

```diff
@@ -1,204 +1,310 @@
+                <img src="portfolioimgs/4.png" alt="github" />
+              </a>
+            </li>
+            <li class="socialMedia">
+              <a class="get-my-contact" href="#">
+                <img src="portfolioimgs/5.png" alt="peace" />
+              </a>
+            </li>
+          </ul>
         </div>
-<div class="resume">
-  <button type="button">Get my resume</button>
-  <ul class="languages">
- <li>Languages</li>
- <li><img class="scrollDown" src="portfolioimgs/down.png" alt="down">
-</li>
-  </ul>
-
-  <ul class="html-css">
-    <li class="html-container">
-   <img  class ="html" src="portfolioimgs/HTML.png" alt="html"/> <span class="html-li">HTML</span>
-    </li>
-  <li class="css-container">
-   <img class="css" src="portfolioimgs/CSS.png" alt="css"/> <span class="css-li">CSS</span>
-  </li>
-  </ul>
-  <ul class="Frameworks">
-    <li>Frameworks</li>
-   <li> <img class="scrollDown2" src="portfolioimgs/down2.png" alt="down">
-  </li>
-  </ul>
-
-   <ul class="Skills">
-    <li>Skills</li>
-    <li><img class="scrollDown2" src="portfolioimgs/down2.png" alt="down"></li>
-  
-   </ul>
-
-</div>
+        <div class="resume">
+          <button type="button">Get my resume</button>
+          <ul class="languages">
+            <li>Languages</li>
+            <li>
+              <img class="scrollDown" src="portfolioimgs/down.png" alt="down" />
+            </li>
+          </ul>
 
-</section>
+          <ul class="html-css">
+            <li class="html-container">
+              <img class="html" src="portfolioimgs/HTML.png" alt="html" />
+              <span class="html-li">HTML</span>
+            </li>
+            <li class="css-container">
+              <img class="css" src="portfolioimgs/CSS.png" alt="css" />
+              <span class="css-li">CSS</span>
+            </li>
+          </ul>
+          <ul class="Frameworks">
+            <li>Frameworks</li>
+            <li>
+              <img
+                class="scrollDown2"
+                src="portfolioimgs/down2.png"
+                alt="down"
+              />
+            </li>
+          </ul>
 
-    
+          <ul class="Skills">
+            <li>Skills</li>
+            <li>
+              <img
+                class="scrollDown2"
+                src="portfolioimgs/down2.png"
+                alt="down"
+              />
+            </li>
+          </ul>
+        </div>
+      </section>
 
-</div>
-</body>
-</html>
\ No newline at end of file
+      <section class="contact-me">
+          <h5 class="section-header">Contact me</h5>
+          <p class="text-contact">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form>
+            <input
+              type="text"
+              name="name"
+              value="Yeremias"
```

Comment:
- [ ] Kindly change this `value` property into a `placeholder` property. The difference here, it's that the `value` property, sets a default value into your `input` file, and you need to delete it before you can fill the `input` with your own information, but the `placeholder` property sets a "fake" value that will be deleted automatically once you click (or tap) into that `input` field.